src/components/FirstTimeUserFlow/PersonalInfoForm.js

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import {
+	FormControlLabel,
+	FormControl,
+	FormLabel,
+	RadioGroup,
+	Radio,
+	Select,
+	MenuItem,
+	Box,
+	Chip,
+} from '@mui/material/';
+
+import { getAuthUser } from '@/firebase/userDb';
+
+const allColleges = ['Revelle', 'Muir', 'Marshall', 'Warren', 'Roosevelt', 'Sixth', 'Seventh', 'Eight'];
+const allYears = ['Freshmen', 'Sphomore', 'Junior', 'Senior', 'Master', 'PHD'];
+const allMajors = ['Math', 'Computer Science', 'Biology'];
```

Comment:
can we either
- add a TODO comment so we remember to update this with all majors
- or just add the list of all majors right now?